src/lib/swapper/swappers/CowSwapper/getCowSwapTradeQuote/getCowSwapTradeQuote.ts

```diff
@@ -103,9 +103,7 @@ export async function getCowSwapTradeQuote(
   })
 
   // don't show buy amount if less than min sell amount
-  const isSellAmountBelowMinimum = bnOrZero(normalizedSellAmountCryptoBaseUnit).lt(
-    minimumCryptoBaseUnit,
-  )
```

Comment:
fixes cow swapper returning an amount when the input is 0. The logic here was including the hardcoded minimum when it should be using the inputted amount